mesop/utils/host_util.py

```diff
@@ -0,0 +1,11 @@
+import socket
+
+
+def get_default_host() -> str:
+  """
+  This function returns the default host (which is externally accessible)
+  based on the availability of IPv6.
+  """
+  if socket.has_ipv6:
```

Comment:
I was reading through https://stackoverflow.com/questions/66246308/detect-if-ipv6-is-supported-os-agnostic-no-external-program and also this link https://github.com/urllib3/urllib3/pull/611 from the stackoverflow.

Is this an issue for our use case?

Reply:
good catch. fixed. PTAL.